### ISSUE TITLE
Diaphragmatic rupture

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11520]
**Preferred term label:**

**Synonyms**
Ruptured diaphragm

**Definition (free text, please give PubMed ID)**
PMID: 26765342 A 9-year-old gir**l ruptured her diaphragm** due to postoperative vomiting. She subsequently developed a paraesophageal hernia with incarceration of the stomach that was further complicated by the occurrence of a large abdominal hematoma after reduction.

PMID: 39641471 In childhood, there have been reports of bladder diverticula resulting in spontaneous bladder rupture (P1 aged 5 and P3 aged 5 in Malfait et al. (2004)), rectal prolapse (siblings P1 and P2 in Van Damme et al. (2016)), and **diaphragmatic rupture** following vomiting resulting in paraoesophageal hernia with stomach incarceration and an abdominal hematoma (P1 aged 9 in Malfait et al).

PMID: 28306229  Minor criteria Dermatosparaxis EDS (dEDS
1. Soft and doughy skin texture
2. Skin hyperextensibility
3. Atrophic scars
4. GJH25
5. Complications of visceral fragility
(e.g., bladder rupture, **diaphragmatic rupture**, rectal prolapse)
6. Delayed motor development26
7. Osteopenia

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Dermatosparaxis Ehlers-Danlos syndrome ORPHA:1901

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001468
- label: Spontaneous diaphragmatic rupture
- parents: [Abnormality of the diaphragm (HP:0000775)](https://hpo.jax.org/app/browse/term/HP:0000775)
- pmid: PMID:22524904|PMID:41356277|PMID:28785503
- def: A full thickness tear of the diaphragm without significant trauma.
- comment: Spontaneous diaphragmatic rupture is a rare complication of Ehlers-Danlos syndrome in which abnormal collagen synthesis results in tissue fragility and weakness of the diaphragm, predisposing to full-thickness tearing without significant trauma, typically triggered by a sudden increase in intra-abdominal pressure such as coughing, vomiting, or physical exertion, and resulting in herniation of abdominal contents into the thoracic cavity.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11520
- synonyms:
